config/local.example.php

```diff
@@ -9,6 +9,7 @@
     'api_url' => 'http://0.0.0.0:1234',
     'elastic_servers' => ['http://elife_search_elasticsearch:9200'],
     'annotation_cache' => false,
+    'elastic_logging' => true,
```

Comment:
ok for turning it off on `end2end` and `prod` which are going to be the ones with lots of data in them